_component_design/tooltip.md

```diff
@@ -13,28 +13,292 @@ last-modified: 2017-08-17
+
+Tooltips are composed of two primary portions, the tooltip container and the trigger:
+
+- The trigger is the element that displays the tooltip component when interacted with from a user. (see trigger types section)
+
+- The container houses the border, pointer, and content. 
+
+The tooltip will stand out from the background by using a lighter background colour than the interface background, a rule or line around the tooltip box, and a shadow around the tooltip so it appears raised from the items below.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip composition."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-composition.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Tooltip specifications
+
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Spacing
+{: .hxSubSectionTitle}
+Tooltips have varying widths based on the amount of content, however, a tooltip relies on the following consistent spacing specifications:
+
+- 8px between vertical elements
+- 12px between unrelated elements 
+- max-width of 400px
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Spacing specifications for tooltip component."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-spacing-1.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Text values
+{: .hxSubSectionTitle}
+
+- 14px Roboto Medium for headers
+- 14px Roboto Regular for body copy
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Text specifications for tooltip component."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-spacing-2.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Pointer and placement
+{: .hxSubSectionTitle}
+
+- The pointer is 16 px wide at the base.
+- The apex is 8 px from the side of the rectangle. 
+- The pointer should start 12 px from the corner.
+- The side from which the pointer extends depends on the location of the trigger on the screen. Best practice pointer placement relies up the following:
+
+	- Always have the tooltip protrude toward the center of the screen.
+	- Ensure the tooltip does not hide important information.
+	- The tooltip content should be repositioned so that it doesn’t extend off the boundaries of the current screen.
+
+- The depth and width of the remaining side of the tooltip is dynamic based on how large the box needs to be with a max-width of 400px. 
```

Comment:
Remove hyphen between `max` and `width`, and add a space between the number and the unit.

Reply:
updated.